src/index.js

```diff
@@ -15,64 +15,120 @@ if (isNaN(pollInterval)) {
 
 
 const metrics = {
-	first_contentful_paint: new Prometheus.Gauge({
-		name: 'pagespeed_first_contentful_paint_milliseconds',
+	desktop_first_contentful_paint: new Prometheus.Gauge({
+		name: 'desktop_pagespeed_first_contentful_paint_milliseconds',
```

Comment:
I wouldn't rename the existing metrics as that'd break existing reports etc

Reply:
Excellent point, didn't even think of that. Oops!